ui/src/app/components/gke-app-configuration-panel.spec.tsx

```diff
@@ -68,14 +72,13 @@ describe(GKEAppConfigurationPanel.name, () => {
   });
 
   const DEFAULT_PROPS: GkeAppConfigurationPanelProps = {
-    type: AppType.RSTUDIO,
+    appType: AppType.RSTUDIO,
     workspaceNamespace: 'aou-rw-1234',
     onClose: jest.fn(),
     initialPanelContent: null,
+    creatorFreeCreditsRemaining: 300,
```

Comment:
this was null, which caused a spinner, which caused the spinner-detection logic to see an unwanted spinner